templates/ecmonsen-emorypipeline-ci.yaml

```diff
@@ -27,56 +27,6 @@ Resources:
             Action: "s3:GetObject"
             Resource: !Sub "arn:aws:s3:::${LambdaArtifactsBucket}/*"
 
-  CfnDeployerServiceUser:
-    Type: 'AWS::IAM::User'
-  CfnDeployerServiceUserAccessKey:
-    Type: 'AWS::IAM::AccessKey'
-    Properties:
-      UserName: !Ref CfnDeployerServiceUser
-  CfnDeployerServiceRole:
```

Comment:
cannot remove this due to a deployed lambda in the account that contains a hard  dependency on this resource, https://github.com/Sage-Bionetworks/ecmonsen-emorypipeline-lambda-code/blob/master/templates/base.yaml

Reply:
We are now [removing the entire emory pipeline](https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/615) which should resolve this problem and allow us to merge this PR without problems